Components/Platform.py

```
@@ -79,6 +79,17 @@ def move_z_axis(self, distance):
 
         try:
             while current_position < target_distance:
+                if direction_up:
+                    if self.platform.endstop_max.change_detected():
+                        self.platform.ncoder.set_position(self.platform.maks_height)
+                        self.log(f"Endstop MAX triggered. Position set to {self.platform.maks_height} cm.", "warning")
```

Pylint (3.3.7)

[convention] Line too long (118/100)

(C0301)